pkg/util/merr/errors.go

```diff
@@ -124,6 +124,14 @@ var (
 	ErrInvalidSearchResult       = newMilvusError("fail to parse search result", 1805, false)
 	ErrCheckPrimaryKey           = newMilvusError("please check the primary key and its' type can only in [int, string]", 1806, false)
 
+	// mq related
+	ErrDenyProduceInBackup  = newMilvusError("deny to write the message to mq in backup instance", 1900, false)
```

Comment:
1. The mq related errors are at line 104
2. Looks like these error shouldn't be merr, these errors should be showed to users, many of them could be replaced with `ErrIoFailed`

Reply:
I will put these changes into the next kind improvement pr